contracts/contracts/strategies/uniswap/UniswapV3LiquidityManager.sol

```diff
@@ -15,22 +14,9 @@ import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.s
 contract UniswapV3LiquidityManager is UniswapV3StrategyStorage {
     using SafeERC20 for IERC20;
 
-    /***************************************
-            Deposit/Withdraw
-    ****************************************/
-    function _depositAll() internal {
-        UniswapV3Library.depositAll(
-            token0,
-            token1,
-            vaultAddress,
-            poolTokens[token0].minDepositThreshold,
-            poolTokens[token1].minDepositThreshold
-        );
-    }
-
     // TODO: Intentionally left out non-reentrant modifier since otherwise Vault would throw
     function withdrawAssetFromActivePosition(address _asset, uint256 amount)
-        external
+        public
```

Comment:
Does this work on fork? Since it has a onlyVault modifier, this might not work. We could probably have an `internal` function without any modifier and one `external` with necessary modifiers?